ui/src/app/views/workspace/component.ts

```diff
@@ -8,6 +9,29 @@ import {RepositoryService} from 'app/services/repository.service';
 import {User} from 'app/models/user';
 import {UserService} from 'app/services/user.service';
 
+class Notebook {
+  constructor(public name: string, public description: string, public url: string) {}
+}
+class CohortNameFilter implements StringFilter<Cohort> {
```

Comment:
I think its behavior is self-explanatory, but its purpose isn't. I'm looking for a class doc like "Search filter used by the XYZ widget, to determine which of the Cohorts loaded into client-side memory are displayed."